### PR TITLE
feat: create separate package.json for cjs module on build time

### PR DESCRIPTION
**Pull request by AhyoungRyu**
Addresses https://sendbird.slack.com/archives/G01290GCDCN/p1689744629102179?thread_ts=1689719295.903759&cid=G01290GCDCN

Screenshot 2023-07-19 at 3 16 12 PM

### CODE REVIEW

File: scripts/post_build.js [hunk 1]
```diff
@@ -50,7 +50,23 @@ function movePackageJSON() {
...
 }
 
+function copyCJSPackageJSON() {
+  const packageJSONDistPath = path.resolve(getDistPath('/cjs'), 'package.json');
+  fs.writeFileSync(
+    packageJSONDistPath,
+    JSON.stringify({
+      ...packageTemplate,
+      main: 'index.js',
+      type: 'commonjs',
```
Comment (AhyoungRyu) [inline scripts/post_build.js:60]: @sravan-s Is there anything we want to add or omit here?

Comment (sravan-s) [reply scripts/post_build.js:60]: Maybe (?) Even tho they shouldnt do anything ~
    peerDependencies,
    dependencies,
    devDependencies,
    browserslist,

Comment (AhyoungRyu) [reply scripts/post_build.js:66]: https://github.com/sendbird/sendbird-uikit-react/pull/687/commits/61ae6c3684445ba1e04559cac06c20e7e97560ce How about this? 
```
{
  "version": "3.6.3",
  "name": "@sendbird/uikit-react",
  "description": "Sendbird UIKit for React: A feature-rich and customizable chat UI kit with messaging, channel management, and user authentication.",
  "keywords": [
    "sendbird",
    "uikit",
    "react",
    "chat",
    "messaging",
    "real-time",
    "UI components",
    "user authentication",
    "channel management",
    "SDK integration",
    "customizable",
    "feature-rich",
    "social app",
    "customer support",
    "API"
  ],
  "type": "commonjs",
  "main": "index.js",
  "style": "dist/index.css",
  "typings": "index.d.ts",
  "module": "index.js",
  "repository": {
    "type": "git",
    "url": "git+git@github.com:sendbird/sendbird-uikit-react-sources.git"
  },
  "author": "SendBird <support@sendbird.com>",
  "license": "SEE LICENSE IN LICENSE.md",
  "bugs": {
    "url": "https://community.sendbird.com"
  },
  "homepage": "https://sendbird.com",
  "peerDependencies": {
    "react": "^16.8.6 || ^17.0.0 || ^18.0.0",
    "react-dom": "^16.8.6 || ^17.0.0 || ^18.0.0"
  },
  "dependencies": {
    "@sendbird/chat": "^4.9.2",
    "@sendbird/uikit-tools": "0.0.1-alpha.40",
    "css-vars-ponyfill": "^2.3.2",
    "date-fns": "^2.16.1",
    "dompurify": "^3.0.1",
    "prop-types": "^15.7.2"
  },
  "browserslist": [
    "defaults",
    "IE 11"
  ]
}
```

**Review by sravan-s — APPROVED** [review]

File: scripts/post_build.js (hunk 1 above)
Comment (bang9) [reply scripts/post_build.js:67]: AFAIK, we just need one line of package.json `{ type: "commonjs" }`
https://nodejs.org/dist/latest-v18.x/docs/api/packages.html#determining-module-system
image

**Pull request merged**